package.json

```diff
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "@juggle/resize-observer": "^3.0.2",
-    "@momentum-ui/react": "^23.2.1",
+    "@momentum-ui/react": "^23.14.0",
```

Comment:
Seems like the latest version is [23.15.3](https://www.npmjs.com/package/@momentum-ui/react), we should upgrade to that